assets/sponsors.json

```diff
@@ -0,0 +1,28 @@
+{
+    "Platinum":
+        [{
+            "alttext":"sponsor1-border",
+            "imgmicro":"img/micro.png"
+        },
+        {
+            "alttext":"sponsor2-border",
+            "imgmicro":"img/micro.png"
+        },
+        {
+            "alttext":"sponsor3-border",
+            "imgmicro":"img/micro.png"
+        }],
```

Comment:
put square bracket in separate line.

Reply:
Follow the indentation style for json similar to https://github.com/pythonindia/inpycon2018/blob/master/js/timeline.js